src/oraWrapper.ts

```diff
@@ -0,0 +1,24 @@
+import * as ora from "ora";
```

Comment:
Same comment as https://github.com/firebase/firebase-tools/pull/1429#discussion_r297280452 on the parallel PR.

Reply:
Done